packages/core/src/transports/initialize.ts

```diff
@@ -128,6 +128,7 @@ export function initializeTransports(
       } else {
         send([item]);
       }
+      transports.filter((t) => !t.isBatched()).forEach((t) => t.send(item));
```

Comment:
Q: I don't really get the purpose of this `send()` chain.

Can you help to clear the dust 🙈

So:
1. If bacthing is enabled we add an item to the executor: `batchExecutor.addItem(item);`
2. Else we send a single item array
3. Finally, indipendent of 1 and 2 we get all non-batch transport and call `send()`

3.1 we loop over all batched transports, which means nothing get send because in 3. we filter all non-batched, then call send and filter for batched items.